### PR TITLE
add evaluation results screen

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [Create Evaluation Results screen#352](https://github.com/nerubia/kh360-react/issues/352)

BE PR: [update api for evaluation results screen #356](https://github.com/nerubia/kh360-nodejs/pull/356)

### CODE REVIEW

**Review by chadiegilnerubia — APPROVED** [review]
Good :1st_place_medal:

File: src/features/evaluation-results/[id]/evaluation-results-footer.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { useNavigate } from "react-router-dom"
+import { Button } from "../../../components/ui/button/button"
+import { Icon } from "../../../components/ui/icon/icon"
+
+export const EvaluationResultsFooter = () => {
+  const navigate = useNavigate()
+
+  const handleGoBack = () => {
+    navigate(`/my-evaluations`)
```
Comment (jatnerubia) [inline src/features/evaluation-results/[id]/evaluation-results-footer.tsx:9]: I think this should redirect to the evaluation results list screen 🤔

File: src/features/evaluation-results/[id]/evaluation-results-comments.tsx
Comment (jatnerubia) [inline src/features/evaluation-results/[id]/evaluation-results-comments.tsx:1]: Shouldn't this be named 'view'? 🤔

File: src/features/evaluation-results/[id]/evaluation-results-footer.tsx
Comment (jatnerubia) [inline src/features/evaluation-results/[id]/evaluation-results-footer.tsx:1]: Shouldn't this be named 'view'? 🤔

File: src/features/evaluation-results/[id]/evaluation-results-header.tsx
Comment (jatnerubia) [inline src/features/evaluation-results/[id]/evaluation-results-header.tsx:1]: Shouldn't this be named 'view'? 🤔

File: src/features/evaluation-results/[id]/evaluation-results-table.tsx
Comment (jatnerubia) [inline src/features/evaluation-results/[id]/evaluation-results-table.tsx:1]: Shouldn't this be named 'view'? 🤔

File: src/routes/bod-user-route.tsx
Comment (jatnerubia) [inline src/routes/bod-user-route.tsx:1]: change to CM? 🤔

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes ^^

File: src/routes/bod-user-route.tsx
Comment (emmaacha) [reply src/routes/bod-user-route.tsx:1]: Yep, change for this was made in the other PR, will update this branch, thanks!

**Comment by emmaacha** [comment]
Done ^^

**Review by jatnerubia — APPROVED** [review]
Cool

**Pull request merged**